luno_python/client.py

```diff
@@ -782,7 +770,7 @@ def post_market_order(self, pair, type, base_account_id=None, base_volume=None,
         }
         return self.do('POST', '/api/1/marketorder', req=req, auth=True)
 
-    def send(self, address, amount, currency, description=None, destination_tag=None, external_id=None, forex_notice_self_declaration=None, has_destination_tag=None, is_drb=None, is_forex_send=None, message=None):
+    def send(self, address, amount, currency, description=None, destination_tag=None, external_id=None, forex_notice_self_declaration=None, has_destination_tag=None, is_drb=None, is_forex_send=None, memo=None, message=None):
```

Comment:
Is the order of arguments for these two functions within our control, or controlled by the generator? 

My Python is a little rusty, (and assuming this is public) but if someone is calling this function already with send(x,y,z...,message), when they use this new version they'll be calling `memo` with the `message` arg, right?

Reply:
That was the sdk generator - I hope nobody is using positional arguments with python for optional params and instead should be using named ones but I take your point. However outside of changing the sdkgen (and it appears like use of this client) I'm not sure exactly what I can do as it appears to be ordering parameters alphabetically